### ISSUE TITLE
rmdup failure

### ISSUE THREAD

**Issue by ekg** [samtools/samtools#393]
```text
 % samtools rmdup out.bam final.bam                       
[bam_rmdup_core] processing reference NC_000913.3...
Samtools-htslib-API: bam_get_library() not yet implemented
[1]    8277 abort      samtools rmdup out.bam final.bam
```

I'm perplexed. This just worked recently (<1 month ago).

**Comment by fabioticconi** [comment]
Probably you have upgraded to 1.x. I've had to revert to 0.1.20 for now, the last release to have a working rmdup.

For reference: https://github.com/samtools/samtools/issues/291

**Comment by jkbonfield** [comment]
I think the general conclusion has been that Samtools rmdup was inferior to Picard's and Biobambam's implementations anyway (alas I forget the specific reasons why).  One day it'll reappear as markdup, with a fixed algorithm.

It's fair to comment though that rmdup is broken.  We should just remove the subcommand until a suitable reimplementation exists.

**Comment by nodrogluap** [comment]
As much as the method may be technically inferior, its strategy actually works better for my existing genotyping pipelines (reduces false negatives), so please don't get rid of it!

**Comment by mp15** [comment]
The issue is that it's not assessing reads as a pair which means it's removing a lot of reads that aren't PCR duplicates. What filters are you using in your genotyping pipeline? I wonder if it's coincidentally reducing strand bias or similar?

**Comment by jmarshall** [comment]
This was fixed by the commit above (PR #461, in particular 13de48f8dd6bcd71277128143fadc9ab3aa50576); an `rmdup` command that works as well as it ever did will reappear in the upcoming 1.3 release.
